### PR TITLE
Additions of few papers for my experimental paper

### PR DESCRIPTION
**Pull request by jonaskuResearch**
Includes a few new papers of the introduction of my experimental paper. I have added Guillermo, so that he can get some reviewing experience on a small pull request.

### CODE REVIEW

File: bibliography.bib [hunk 1]
```diff
@@ -7082,4 +7082,63 @@ @Article {CarMilHir-etal2020ALR
...
+    number       = {54},
+    year         = {2022},
+    pages        = {1--9},
+    doi          = {},
+    note         = {},
+    annote       = {},
+}
+
+@Article {BerHalHsiKum2009IEEETR,
+    title        = {Optimized Stochastic Policies for Task Allocation in Swarms of Robots},
```
Comment (dagarzonr) [inline bibliography.bib:7119]:
```suggestion
    title        = {Optimized stochastic policies for task allocation in swarms of robots},
```

File: bibliography.bib [hunk 2]
```diff
@@ -7082,4 +7082,63 @@ @Article {CarMilHir-etal2020ALR
...
+    number       = {4},
+    year         = {2009},
+    pages        = {927--937},
+    doi          = {10.1109/TRO.2009.2024997},
+    note         = {},
+    annote       = {},
+}
+
+@Article {ProHsiKum2017IEEETR,
+    title        = {The Impact of Diversity on Optimal Control Policies for Heterogeneous Robot Swarms},
```
Comment (dagarzonr) [inline bibliography.bib:7133]:
```suggestion
    title        = {The impact of diversity on optimal control policies for heterogeneous robot swarms},
```

**Review by dagarzonr — COMMENTED** [review]
Just added a few changes on capitalization.

Could you also take a look to the rules of proceedings.bib and proceedings-short.bib.

The rule states:

 @comment{It is sorted alphabetically by the string definition of the proceedings name, including every letter. Space goes before other letters.}

Either we sort again the proceedings with respect to the name of the proceedings or we correct the rule. 

I think they keys here are more consistent than the names of the proceedings, but we recently reorder journal.bib alphabetically by the name. I am fine with either sorting.

Could you also please add a rule so that proceedings-short.bib mirrors proceedings.bib. (like in the last PR of journal.bib)?

**Review by glegarda — COMMENTED** [review]
Other than the capitalisation changes suggested by @dagarzonr, I could not find anything else worth noting.

Regarding the sorting rules, I agree with @dagarzonr in that sorting by key would be a better option, as I find it more intuitive to search by key than by title (e.g., no "The" to be considered when searching by key).

File: bibliography.bib [hunk 3]
```diff
@@ -3787,7 +3787,7 @@ @InProceedings {UsuAri2003arob
     annote      = {},
 }
 
-@Article {StaRis2002EC,
+@Article {StaMik2002EC,
```
Comment (dagarzonr) [inline bibliography.bib:3790]:
```suggestion
@Article {StaMii2002EC,
```

**Review by dagarzonr — COMMENTED** [review]
I added a correction to the key of NEAT

**Pull request merged**